gdal/ogr/ogrsf_frmts/mitab/mitab_ogr_driver.cpp

```diff
@@ -208,6 +208,7 @@ void RegisterOGRTAB()
     poDriver->SetMetadataItem(GDAL_DS_LAYER_CREATIONOPTIONLIST,
 "<LayerCreationOptionList>"
 "  <Option name='BOUNDS' type='string' description='Custom bounds. Expect format is xmin,ymin,xmax,ymax'/>"
+"  <Option name='ENCODING' type='string' description='to override the encoding interpretation of the DAT/MID with any encoding supported by CPLRecode or to \"\" to avoid any recoding (Neutral charset)'/>"
```

Comment:
Should be documented in drv_mitab.html (with mention it is available since GDAL 2.3)